pom.xml

```diff
@@ -408,7 +413,7 @@
         <carbon.analytics.version>2.0.197</carbon.analytics.version>
         <carbon.analytics.version.range>[2.0.197, 3.0.0)</carbon.analytics.version.range>
         <!--Analytics common-->
-        <carbon.analytics-common.version>6.0.36</carbon.analytics-common.version>
+        <carbon.analytics-common.version>6.0.40-SNAPSHOT</carbon.analytics-common.version>
```

Comment:
@lasanthaS can you add the released version. carbon-analytics-common 6.0.41 is released.